doc/source/cluster/kubernetes/configs/static-ray-cluster-networkpolicy.yaml

```diff
@@ -0,0 +1,309 @@
+# If your Kubernetes has a default deny network policy for pods, you need to manually apply this network policy 
+# to allow the bidirectional communication among the head and worker nodes in the Ray cluster.
+
+# Ingress
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: ray-cluster-ingress
+spec:
+  podSelector:
+    matchLabels:
+      app: ray-cluster-head
+  policyTypes:
+    - Ingress
+  ingress:
+  - from:
+    - podSelector: {}
+    ports:
+    - protocol: TCP
+      port: 6379
+  - from:
+    - podSelector: {}
+    ports:
+    - protocol: TCP
+      port: 6380
+  - from:
+    - podSelector: {}
+    ports:
+    - protocol: TCP
+      port: 52365
+  - from:
+    - podSelector: {}
+    ports:
+    - protocol: TCP
+      port: 8265
+  - from:
+    - podSelector: {}
+    ports:
+    - protocol: TCP
+      port: 8076
```

Comment:
can we simplify the range here? https://kubernetes.io/docs/concepts/services-networking/network-policies/#targeting-a-range-of-ports

Reply:
I thought about this before. I listed all the ports here mainly for backward compatibility. The range port feature is introduced for the newer version of K8S API, which is not supported in the old versions.

Reply:
Done, I added more contexts [here](https://github.com/ray-project/ray/pull/31403/commits/e12a4780801944cb58c7114182822f43c7fa0a1b#diff-faf3cabc07b24c76a81c1a25fb2b5a5ad1772dd11dfdb511396073c5639d9918R4).